src/lightgbm/src/main/scala/LightGBMClassifier.txt

```diff
@@ -0,0 +1,2 @@
+Trains a LightGBM Binary Classification model, a fast, distributed, high performance gradient boosting
```

Comment:
Maybe add a link to the GitHub page for LightGBM?

Reply:
done